apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1581,29 +1588,39 @@ acc_txs_test(Pubkey, Offset, Limit) ->
                             Tx = aetx_sign:tx(SignedTx),
                             lists:member(PKDecoded, aetx:accounts(Tx))
                         end,
-                    {ok, AllPendingTxs} = rpc(aec_tx_pool, peek, [infinity]),
-                    FilteredPendingTxs =
-                        lists:filter(
-                            fun(Tx) ->
-                                PubkeyInTx(Tx) andalso
-                                apply_tx_type_filter(TT, [Tx]) =/= []
-                            end,
-                            AllPendingTxs),
-                    AccountPendingTx = encode_pending_tx(
-                                         lists:sort(fun(A, B) ->
-                                                        aetx:nonce(aetx_sign:tx(A)) >=
-                                                        aetx:nonce(aetx_sign:tx(B))
-                                                    end,
-                                                    FilteredPendingTxs),
-                                        TxEncoding),
+                    AccountPendingTx =
+                        case ShowPending of
+                            false -> [];
+                            _ ->
```

Comment:
Please no open clause here - it makes test more fragile and less readable (I see clue is `[default, true, false]` in caller function).